lib/src/includes/signed_video_sign.h

```diff
@@ -337,12 +337,12 @@ signed_video_set_end_of_stream(signed_video_t *self);
  *     // Handle error
  *   }
  *   // Apply any configuration of choice.
+ *   // All necessary configurations need to be done prior to this call.
```

Comment:
Now you have two lines saying roughly the same thing. I think you can remove the second one.